deploy-gae-build-task/deploy-gae.ts

```diff
@@ -79,18 +84,17 @@ export async function deployGae({
           .line('beta app deploy --quiet --verbosity=info')
```

Comment:
You you can remove the `beta` channel from here, deploying is now fully supported in the GA track.

Reply:
Being done in another PR.